src/test/unit/flight_imu_unittest.cc

```diff
@@ -76,6 +80,15 @@ extern "C" {
 
 const float sqrt2over2 = sqrtf(2) / 2.0f;
 
+void quaternion_from_axis_angle(quaternion* q, float angle, float x, float y, float z) {
+    fpVector3_t a = {{x, y, z}};
+    vectorNormalize(&a, &a);
+    q->w = cos(angle / 2);
+    q->x = a.x * sin(angle / 2);
```

Comment:
I'm sure compiler will do this anyway, but shouldn't we precalculate `sin(angle / 2)`?